README.md

```diff
@@ -1431,6 +1431,7 @@ _Related: [Wikis](#wikis)_
 - [Stikked](https://github.com/claudehohl/Stikked) - Advanced and beautiful pastebin. `GPL-3.0` `PHP`
 - [Sup3rS3cretMes5age](https://github.com/algolia/sup3rS3cretMes5age) - Very simple (to deploy and to use) secret message service using Hashicorp Vault as a secrets storage. `MIT` `Go`
 - [wantguns/bin](https://github.com/wantguns/bin) - Minimal pastebin for both textual and binary files shipped in a single statically linked binary. ([Demo](https://basedbin.fly.dev)) `GPL-3.0` `Rust`
+- [Wastebin](https://github.com/matze/wastebin) - Lightweight, minimal and fast pastebin with an SQLite backend. ([Demo](https://wastebin-pkue.onrender.com/)) `MIT` `Rust`
```

Comment:
The demo site looks down to me, also looks down to https://downforeveryoneorjustme.com/wastebin-pkue.onrender.com?proto=https, please remove the demo link

Reply:
It's not down but the first request after 15 minutes of inactivity can take "up to 30 seconds" to spin up the instance again. It's a caveat with render.com's free plan. If you try now, it's back again.

Reply:
Anyhow, removed the link.